lib/metered_raven_error_handler.rb

```diff
@@ -8,7 +8,7 @@ def call(ex, context)
 
       retry_count = (context['retry_count'] || (context[:job] && context[:job]['retry_count'])).try(:to_i)
 
-      if retry_count.nil? || retry_count > 1
+      if !retry_count.nil? && retry_count > 0
```

Comment:
Should we make this count configurable?

Reply:
This seems like something we should do eventually, but maybe not right now since it will require figuring out the right way to get the config set up?